### PR TITLE
Addressing #12

### PR DESCRIPTION
**Pull request by ldecicco-USGS**

### CODE REVIEW

**Comment by ldecicco-USGS** [comment]
@rbcarvin have a look at this and see if it's better

**Comment by rbcarvin** [comment]
@ldecicco-USGS I get the same error as before.

**Comment by ldecicco-USGS** [comment]
And you used my branch like this?:
```
git checkout -b ldecicco-USGS-master master
git pull https://github.com/ldecicco-USGS/Rainmaker.git master
```
then rebuilt?

**Comment by rbcarvin** [comment]
I got to your branch (using the first line in your comment) but the pull didn't work. "unable to get local issuer certificate"

**Comment by ldecicco-USGS** [comment]
Oh yeah, classic mistake:
```
git checkout -b ldecicco-USGS-master master
git pull git://github.com/ldecicco-USGS/Rainmaker.git master
```

**Comment by rbcarvin** [comment]
That worked. Thanks.

**Comment by rbcarvin** [comment]
I'm now trying to run this with the Cooperator data (in the package data folder), and am running into an issue. 
```r
PrecipEvents <- RMevents_sko(df = PrecipPrep,ieHr = 2,rainthresh = 0,rain = "rain",time = "pdate")
Error in if (dif_time[[i - 1]] >= ieMin) { : 
  missing value where TRUE/FALSE needed
```

File: R/RMEvents_sko.R [hunk 1]
```diff
@@ -15,6 +15,14 @@
 #' @examples
 RMevents_sko <- function(df,ieHr=6,rainthresh=5.1,rain="rain",time="pdate"){
```
Comment (ldecicco-USGS) [inline R/RMEvents_sko.R]: @rbcarvin : I added 2 checks here. 1 is to make sure the `time` column is correctly defined. The other is to check if all the dates are NA. How about you merge this pr and try to do a little debugging on the cooperator data to try to figure out why it's not working.

**Comment by rbcarvin** [comment]
egads, its the Timezone thing introducing NA's. When I convert to UTC during the RMprep function, this issue goes away...BUT...the times are off by 5 hours. Is there an easy way to get this to switch the time back to UTC-5:00? either at the very end of the workflow, or as a step in RMevents?
```r
Raw.coop <- (read.csv("D:/RCode/Rainmaker/data/Cooperator.csv"))
PrecipPrep <- RMprep(df = Raw.coop,prep.type = 3,date.type = 4,cnames.in = "VALUE")
PrecipEvents <- RMevents_sko(df = PrecipPrep,ieHr = 2,rainthresh = 0.02,rain = "rain",time = "pdate")
PrecipEventList <- PrecipEvents$storms2
StormSummary <- RMIntense(df = PrecipPrep,date = "pdate",rain = "rain",df.events = PrecipEventList,
                          sdate = "StartDate",edate = "EndDate",depth = "rain",xmin=c(5,10,15,30,60))
```

**Comment by limnoliver** [comment]
Bug seems to be issue with time conversion. Likely that original package also had issues when logger did not account for dst, but now RMevents relies on dates, so has missing values when dst kicks in.

**Pull request merged**